### PR TITLE
Update project urls in setup cfg to include Twitter and Slack

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -5,8 +5,11 @@ url = https://github.com/alteryx/featuretools
 download_url = https://pypi.org/project/featuretools/
 project_urls = 
     Documentation = https://featuretools.alteryx.com
-    Source = https://github.com/alteryx/featuretools
-    Tracker = https://github.com/alteryx/featuretools/issues
+    Source Code = https://github.com/alteryx/featuretools/
+    Changes = https://featuretools.alteryx.com/en/latest/release_notes.html
```
Comment (thehomebrewnerd) [inline setup.cfg:9]: Do we want this to go to "latest" or "stable"?

File: setup.cfg [hunk 2]
```diff
@@ -45,7 +48,7 @@ install_requires =
     distributed >= 2021.10.0
     dask[dataframe] >= 2021.10.0
     psutil >= 5.6.6
-    click >= 7.0.0
+    click >= 7.0.0, <8.1.0
```
Comment (rwedge) [inline setup.cfg:51]: we can unpin click and use black 2.3.0

Comment (gsheni) [reply setup.cfg]: Awesome, done

File: .github/workflows/install_test.yml [hunk 3]
```diff
@@ -8,13 +8,16 @@ on:
...
-    runs-on: ubuntu-latest
+    name: ${{ matrix.os }} - ${{ matrix.python_version }} install featuretools complete
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
         python_version: ["3.7", "3.8", "3.9", "3.10"]
+        exclude:
+          - python_version: "3.10"
+          - os: macos-latest
```
Comment (rwedge) [inline .github/workflows/install_test.yml:19]: needs to be 
```
- python_version: "3.10"
  os: macos-latest
```
if you only want macos with 3.10 excluded

Comment (gsheni) [reply .github/workflows/install_test.yml]: fixed

**Review by rwedge — APPROVED** [review]
LGTM

**Comment by rwedge** [comment]
what's the issue with macos and python 3.10?

**Comment by gsheni** [comment]
@rwedge There seems to be some libomp (or gcc) issue with matrixprofile (which is a tsfresh requirement).

**Pull request merged**